src/bundle/Resources/public/js/scripts/helpers/middle.ellipsis.js

```diff
@@ -1,16 +1,41 @@
 (function (global, doc, ibexa) {
-    const parseAll = () => {
+    const resizeEllipsisObserver = new ResizeObserver((entries) => {
+        entries.forEach((entry) => {
+            parseAll(entry.target);
+        });
+    });
+    const parseAll = (baseElement = doc) => {
+        if (!baseElement) {
+            return;
+        }
+
         const middleEllipsisContainers = [...doc.querySelectorAll('.ibexa-middle-ellipsis')];
 
+        if (baseElement instanceof Element) {
```

Comment:
why do we want default `doc` as baseElement if this will be false for `doc` and `baseElement` is not used in other places?